test/box-luatest/downgrade_test.lua

```diff
@@ -973,3 +973,40 @@ g.test_downgrade_global_names = function(cg)
         t.assert_equals(#box.space._cluster:format(), 2)
     end)
 end
+
+-----------------------------
+-- Check downgrade from 3.1.0
+-----------------------------
+
+g.test_downgrade_func_trigger = function(cg)
```

Comment:
Nit. Please fix indentation in this function.

Reply:
I'm not sure I understand what you're talking about, but I revised the indentation in this function.